contrib/builder/deb/armhf/raspbian-jessie/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM resin/rpi-raspbian:jessie
```

Comment:
No there is no guarantee that it will work correctly, eg we link to some C libraries that might have some uses of non compatible instructions in them. I think we would need an official raspbian image.